src/lib.rs

```diff
@@ -2,6 +2,5 @@
 pub mod cli;
 pub mod client;
 pub mod fanout;
-pub mod proxy;
```

Comment:
Why not keep `proxy` and just remove it from the tower service in the bin? If at some point we want to fanout again this will be useful.